docs/pages/machine-id/workload-identity/aws-roles-anywhere.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Configuring Workload ID and AWS Roles Anywhere
+description: Configuring AWS to accept Workload ID certificates as authentication using AWS Roles Anywhere
+---
+
+Teleport's Workload Identity issues flexible short-lived identities in X.509
+certificates. AWS Roles Anywhere allows you to use these certificates to
+authenticate to AWS services.
+
+This can be useful in cases where a machine needs to securely authenticate with
+AWS services without the use of a long-lived credential. This is because the
+machine can authenticate with Teleport without using any shared secrets by
+using one of our delegated join methods.
+
+This implementation differs from Teleport's Application Access for AWS feature
+in a few ways:
+
+- Requests to AWS are not proxied through Teleport, meaning reduced latency but
+  also less visibility, as these requests will not be recorded in Teleport's
+  audit log.
+- Workload ID works with any AWS client, including the commandline tool but also
+  their SDKs
+- Teleport Application Access for AWS does not work with Machine ID and
+  therefore cannot be used when a machine needs to authenticate with AWS.
+
+Whilst this guide is primarily aimed at allowing a machine to access AWS,
+the `tsh svid issue` command can be used in place of Machine ID to allow a human
+user to authenticate with using AWS Roles Anywhere.
+
```

Comment:
We can turn the text between "This implementation differs" and the "Prerequisites" heading into the "How it works" H2.

Reply:
5a907bba55